app-modules/monitoring/src/Providers/MonitoringServiceProvider.php

```
@@ -0,0 +1,203 @@
+     * When no cache store is configured, leave spatie's in-memory default in place.
+     */
+    private function registerStorage(): void
+    {
+        $store = config('prometheus.cache');
+
+        if ($store === null) {
+            return;
+        }
+
+        $this->app->scoped(CollectorRegistry::class, fn (): CollectorRegistry => new CollectorRegistry(
+            new CacheAdapter(Cache::resolve($store)),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'registerStorage'. (undefined)

(StaticAccess)